docker/annovar.dockerfile

```diff
@@ -9,9 +9,35 @@ USER root
 RUN echo "deb [trusted=yes] http://statgen.us/deb ./" | tee -a /etc/apt/sources.list.d/statgen.list && \
     apt-get --allow-insecure-repositories update && \
     apt-get install -y annotation-tutorial && \
-    apt-get clean && mv /home/shared/functional_annotation/* /home/jovyan && rm -rf /home/shared && chown jovyan.users -R /home/jovyan/*
+    apt-get clean
+
+RUN curl -s -o /usr/local/bin/pull-tutorial.sh https://raw.githubusercontent.com/statgenetics/statgen-courses/master/src/pull-tutorial.sh
+RUN chmod a+x /usr/local/bin/pull-tutorial.sh
+
+# Insert this to the notebook startup script,
+# https://github.com/jupyter/docker-stacks/blob/fad26c25b8b2e8b029f582c0bdae4cba5db95dc6/base-notebook/Dockerfile#L151
+RUN sed -i '2 i \
+    pull-tutorial.sh annovar & \
+    copy-datasets.sh & \
+    '  /usr/local/bin/start-notebook.sh
+
+# Content for copy-datasets.sh script
+RUN echo "#!/bin/bash \n\
+\n\
+mkdir -p /tmp/.datacache \n\
+cd /tmp/.datacache \n\
+\n\
+# Operations specific to this exercise. \n\
+cp -r /home/shared/functional_annotation/* ./ \n\
+ln -s ../humandb /home/jovyan/work/humandb \n\
+\n\
+chown -R jovyan.users * /home/jovyan \n\
+mv * /home/jovyan/work \n\
```

Comment:
@cbroad by doing this, you still move `humandb` to `work`? 

I don't have problem moving it if it is too much trouble otherwise. But if you move it, the ln will not be necessary